mkosi/__init__.py

```diff
@@ -115,9 +116,9 @@
 SomeIO = Union[BinaryIO, TextIO]
 PathString = Union[Path, str]
 
+MKOSI_COMMANDS_NEED_BUILD = (Verb.shell, Verb.boot, Verb.qemu, Verb.serve)
+MKOSI_COMMANDS_SUDO = (Verb.build, Verb.clean, Verb.shell, Verb.boot, Verb.qemu, Verb.serve)
 MKOSI_COMMANDS_CMDLINE = ("build", "shell", "boot", "qemu", "ssh")
```

Comment:
Why not change this list to contain enum values instead of strings? I found a `args.verb not in MKOSI_COMMANDS_CMDLINE` check which will break unless this list is changed to contain enum values.